readme.md

```diff
@@ -230,6 +268,11 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
-```
 
+Instructor.findOne({ name: 'Andy' }, function (err, doc) {
+  if (err) return handleError(err);
+  doc.update({wishlist_items: "Resin Laying Deer Figurine, Gold"})
+})
```

Comment:
Almost...

The biggest thing: you need to include a `doc.save();` in order for your change to persist. In addition, using `.update` would overrwrite any wishlist items already existed for that record. `.push()` would prevent that.